diesel/src/pg/backend.rs

```diff
@@ -39,7 +70,7 @@ impl Queryable<(Oid, Oid), Pg> for PgTypeMetadata {
 impl Backend for Pg {
     type QueryBuilder = PgQueryBuilder;
     type BindCollector = RawBytesBindCollector<Pg>;
-    type RawValue = [u8];
+    type RawValue = RefFamily<PgValue>;
```

Comment:
Why this needs to be `RefFamily<PgValue>`? `PgValue` itself implements the necessary trait. (In your version, if the change above is applied, this should be `PgValueProxy`)